calc.py

```diff
@@ -2,7 +2,19 @@
 import sys
 import time
 
+class NotSupportedVersionException(BaseException):
+    def __init__(self):
+        pass
 #all helper functions
+def versionCheck():
+    MinimumPythonVersion=(3, 10)    
```

Comment:
Did I get it right this time?

Reply:
Yes, looks good to me! Thanks for fixing it